src/gromacs/nblib/tests/nbnxnsetup.cpp

```diff
@@ -48,17 +48,23 @@
 
 #include "testutils/testasserts.h"
 
+#include "testhelpers.h"
+#include "testsystems.h"
+
 namespace nblib
 {
 
 namespace test
 {
-/*
-TEST(NBlibTest, NbfpSetCorrectly)
-{
 
+TEST(NBlibTest, CanConstructNbvSetupUtil)
+{
+    ArgonSimulationStateBuilder argonSimulationStateBuilder;
+    SimulationState             system = argonSimulationStateBuilder.setupSimulationState();
+    NBKernelOptions             options;
+    EXPECT_NO_THROW(auto nbvSetupUtil = std::make_unique<NbvSetupUtil>(system, options));
```

Comment:
The "auto nbvSetupUtil = " is not needed.

Reply:
Done.